server/graphql/v2/enum/DateTimeField.js

```diff
@@ -8,5 +8,13 @@ export const DateTimeField = new GraphQLEnumType({
       value: 'createdAt',
       description: 'The creation time of a resource',
     },
+    UPDATED_AT: {
+      value: 'updatedAt',
+      description: 'The update time of a resource',
+    },
+    PUBLISHED_AT: {
+      value: 'publishedAt',
```

Comment:
The approach looks good, my only concern is that we'll get an ugly Sequelize error whenever we try to fetch by `PUBLISHED_AT` on a model that doesn't support it (e.g. `expenses` query). Ideally, I'd love us to be clear on the supported types rather than throwing errors. Something like:

```es6
orderBy: {
  type: new ChronologicalOrderInput({ fields: ['createdAt', 'publishedAt'] }),
  description: 'The order of results',
},
```

Reply:
Ok @Betree , I have created a new ChronologicalWithPublishedAtOrderInput type with a new DateTimeWithPublishedAtField enum. It will be applied only on the Account.updates to prevent any error  => will it be clear enough for you ? I can adapt the name of the input or enum to follow your own convention. Just let me know and I will rename the new input and enum file 👍

Reply:
Personal preference, I would use names to make it explicit that these types are for `Updates`:
- `UpdateChronologicalOrderInput` (input)
- `UpdateDateTimeField` (enum) 

As you mentioned during the call, a shared interface could be a nice to have!